tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/BuilderUtil.scala

```diff
@@ -50,6 +50,17 @@ object BuilderUtil {
     yEx <- y
   } yield unsafeMethod(xEx, yEx)
 
+  /** Lifts a tertiary unsafe method to a TBuilderInstruction method */
+  def tertiaryFromUnsafe(
```

Comment:
I think tertiary is third of rank, whereas ternary is a group of three?

```suggestion
  /** Lifts a ternary unsafe method to a [[TBuilderInstruction]] method */
  def ternaryFromUnsafe(
```

Reply:
No, you're right, I was apparently a bit absentminded here. I can't commit this directly, because it needs refactoring though.